svelte-kit-scss/src/lib/helpers/issues.ts

```diff
@@ -1,28 +1,63 @@
-import type {
-  IssuesAPIResponse,
-  PullRequestItemAPIResponse,
-  RepoIssue,
-  RepoIssues,
+import {
+  type GithubSearchIssueLabel,
+  type GithubSearchIssue,
+  type GithubSearchIssueApiResponse,
+  type Issue,
+  type IssueCollection,
+  type IssueLabel,
+  type IssueUser,
+  type GithubSearchIssueUser,
+  type GithubSearchIssueAssignee,
+  type IssueAssignee,
+  IssueState,
 } from '$lib/interfaces';
 
-export const remapRepoIssue = (item: PullRequestItemAPIResponse): RepoIssue => {
+const remapIssueLabel = (label: GithubSearchIssueLabel): IssueLabel => ({
+  id: label.id,
```

Comment:
What's the difference between a `helper` and a `util`?

So these functions all see to map `GithubSearchIssues*` to `Issue*`, maybe these should live closer to where they are actually used? Like in the issue service?